packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -457,7 +457,7 @@ module.exports = function(log, config, oauthdb) {
 
     const links = this._generateLinks(
       this.verificationUrl,
-      message.email,
+      message,
```

Comment:
A lot of these similar changes - I needed access to the `uid` property down in `_generateLinks` and figured I might as well pass this whole object rather than adding another parameter. Unfortunately, that resulted in this same change to 20 call sites. But, I think it'll be worth it if we ever need any other properties from here